samples/tutorials/Data passing in python components.ipynb

```diff
@@ -5,7 +5,7 @@
    "metadata": {},
```

Comment:
<p>Maybe also add a kfp_endpoint here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
This cell was unused - removed it.

Reply:
<p>Good catch. Fixed</p>